solana/bridge_token_factory/programs/bridge_token_factory/src/state/config.rs

```diff
@@ -22,4 +22,5 @@ pub struct Config {
     pub max_used_nonce: u64,
     pub derived_near_bridge_address: [u8; 64],
```

Comment:
According to issue 12, the value 64 should be defined as a constant.

Reply:
I don't think it's necessary